ambry-api/src/main/java/com.github.ambry/account/AclService.java

```diff
@@ -34,6 +35,17 @@
    */
   boolean hasAccess(P principal, Resource resource, Operation operation);
 
+  /**
+   * Makes a resource access decision.
```

Comment:
Could you give a small example of the different resolution strategies in the javadoc